invenio_app_ils/literature/search.py

```diff
@@ -21,7 +21,7 @@ def filter_periodical_issues(query_string):
         from distutils.util import strtobool
         from flask import request
 
-        include_all = request.values.get("include_all", "yes")
+        include_all = request.values.get("include_all", "no")
```

Comment:
do we really need at all this extra param `include_all`?

Reply:
We talked about it but I will include it here too for future reference, `include_all` was needed for the frontsite series details page since serials can have both documents and multipart monographs as volumes.